fundamentals/android/auth-sign-in-with-google/Notes/app/src/main/java/com/notes/app/screens/authentication/sign_up/SignUpScreen.kt

```diff
@@ -152,17 +157,23 @@ fun SignUpScreen(
 
         Spacer(modifier = Modifier
             .fillMaxWidth()
-            .padding(4.dp))
+            .padding(8.dp))
 
         Text(text = stringResource(R.string.or), fontSize = 16.sp, color = Purple40)
 
         Spacer(modifier = Modifier
             .fillMaxWidth()
-            .padding(4.dp))
+            .padding(8.dp))
 
         AuthenticationButton(R.string.sign_up_with_google) { credential ->
             viewModel.onSignUpWithGoogle(credential, openAndPopUp)
         }
+
+        LaunchedEffect(Unit) {
+            launchCredManBottomSheet(context) { result ->
+                viewModel.onSignUpWithGoogle(result, openAndPopUp)
+            }
+        }
```

Comment:
nit: can we move this up a few lines (ie. to the beginning of this composable) ? I think it would make it more discoverable - it's harder to spot a `LaunchEffect` if it's at the bottom/end of the composable function.

Reply:
Great point! I moved it up on both `SignUpScreen` and `SignInScreen`